modules/arcgis/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@deck.gl/arcgis",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "scripts": {},
+  "dependencies": {
```

Comment:
We've had issues with users accidentally installing multiple copies of deck due to nested dependencies. Most submodules these days use `peerDependencies` and `@deck.gl/core`, e.g.
 https://github.com/uber/deck.gl/blob/93c78f3ea32b9267c0c6f297e06c8f7378c73882/modules/mapbox/package.json#L32-L34

Reply:
Thank you @Pessimistress! I think this was manifesting for me as multiple versions of `luma.gl` that would lead to some sort of mismatch. I'll make the change.